### DISCUSSION TITLE
Strange proposal

### DISCUSSION THREAD

**Discussion by Lancetnik** [kpn/kstreams#185] [General]
Hello, @marcosschroh and team! I really like you project and Roadmap (it looks really promising), but what do you think about making the path a bit shorter?

I am not sure about your company goals on the **k streams**, but if it is just a side project for internal usage - I would like you to invite to **FastStream** contribution. We have the pretty close to your features roadmap, but for all potential brokers, not just a Kafka. Also, we already implemented a lot of features you plan, so we can just start from the missed points and make our collaborative work much stronger.

Anyway, I wish you good luck with your project and I will be watching its development. You are strong team and able to make the great thing!

**Comment by marcosschroh** [comment]
Hi @Lancetnik 

Thanks for reaching out! Thanks for your kind words and for the invitation to collaborate with you on `FastStream`.

The idea of `kstreams` was to use it internally, but we saw that nothing was on the market (2 years ago) so we decided to open source it. We will continue working on `kstreams`, mainly because our needs could change at `KPN` and it will be easier for us to change the code based on our requirements (of course always taking into account our external users).

In any case, we have decided to try `FastStream` and of course collaborate with you. Your project is more mature, it has more features and your community is bigger than ours!! We will definitely give it a try!!!

Could you share your `roadmap`? I could not find it.

**Reply by Lancetnik** [reply]
@marcosschroh, sorry, missed this part of the project 😄 
Created [Roadmap](https://github.com/airtai/faststream/issues/1510) for you

**Reply by marcosschroh** [reply]
Thanks for sharing the roadmap! When I have some free time I will try to add some code!
